support/COMMUNITY_CONTACTS.md

```diff
@@ -0,0 +1,78 @@
+# Duty
+
+Every week we dedicate one individual (the community contact) to watch for user
+issues and either answer them or redirect the questions to other who can. The
+community contact's duty (subject to change) is as followed:
+
+## Daily check list
+
+- Check the knative-users@ mailing list and #eventing-questions Slack channel for
+  new unanswered questions.
+- Check
+  [unassigned issues / untriaged issues](https://github.com/knative/eventing/issues?q=is%3Aopen+is%3Aissue+no%3Aassignee+no%3Amilestone)
+  for unanswered questions.
+- You should already be added to @eventing-help usergroup by the previous contact
+  person, and will be tagged into user questions in other channels.
+
+## Weekly check list
+
+### Monday
+
+- Check the [Eventing test grid](https://testgrid.knative.dev/eventing) for
+  flakiness to pick a test and focus on fixing it during your week. Once you
+  pick the test flake, assign the corresponding bug filed by flakiness test
+  reporter to yourself so that others won't pick the same test to fix.
+- Join the @eventing-help user group if you haven't been invited by the previous
+  contact person using this
+  [link](https://app.slack.com/client/T93ELUK42/browse-user-groups/user_groups/S0191HUK2Q0)
+
+### Tuesday - Thursday
+
+Continue on fixing test flakes and answer
+
+### Friday
+
+- Remove yourself from @eventing-help usergroup and add the next contact using
+  this
+  [link](https://app.slack.com/client/T93ELUK42/browse-user-groups/user_groups/S0191HUK2Q0).
+  If you don't have permission, ask in the Slack channel
+  #steering-toc-questions.
+- Email the next contacts, cc'ing knative-dev@ with a short summaries of the
+  user questions encountered and links to them.
+- File issue or document suggested improvements to this process.
+
+## SLO
+
+Participation is voluntary and based on good faith. We are only expected to
+participate during our local office hour.
```

Comment:
This sentence is a bit unclear, wdym by _our_ in this context? What about rephrasing to:

> Participation is voluntary and based on good faith. We expect participants to be available only during their local office hour
participate during our local office hour.

Reply:
+1. Fixed.